model/loss/base_loss.py

```diff
@@ -20,6 +20,8 @@
 import tensorflow as tf
 from deeplab2.model import utils
 
+_DEPTH_IGNORE_LABEL = 0
```

Comment:
Add comment for this.

Seems that we are having _DEPTH_IGNORE_LABEL in multiple files now.
Could we further unify them at one place?
For example, how about we have a helper function, called get_depth_ignore_label, in utils?

Reply:
Sure. But these two files do not have a shared import -- they seem quite independent. Which file do you think we should import?